roles/sugarizer/tasks/main.yml

```diff
@@ -11,14 +11,23 @@
         state=link
 
 - name: Install sugarizer required packages
-  package: name={{ item }} 
+  package: name=nodejs
            state=present
-  with_items:
-    - nodejs
-    - npm
+#    - npm
   when: internet_available 
-  tags:
-    - download
+
+- name: Install npm non debian
+  package: name=npm
+           state=present
+  when: internet_available and not is_debian
+
```

Comment:
the package npm routine would still needed for debian 8 or is this a running change to fix a sugarizer issue?